test/test-integration/src/test/java/org/wso2am/micro/gw/tests/endpoints/ProductionSandboxTestCase.java

```diff
@@ -47,11 +47,11 @@ void start() throws Exception {
         //deploy the api
         //api yaml file should put to the resources/apis/openApis folder
         String prodSandApiZipfile = ApiProjectGenerator.createApictlProjZip(
-                "prod-sand/prodSandOpenAPI.yaml");
+                "prod-sand/prodSand_api.yaml", "prod-sand/prodSand_swagger.yaml");
```

Comment:
Why these resources are being renamed to `swagger`? since the open standard is `openapi` we should use openapi not swagger

Reply:
When an API project is created via apictl, the file inside the Definitions directory is swagger.yaml. Therefore, I used the name 'swagger' to resemble that file.